src/api/rpc_api/author.rs

```diff
@@ -32,7 +34,7 @@ pub trait SubmitExtrinsic {
 
 	/// Submit an extrsinic to the substrate node, without watching.
 	/// Retruns the extrinsic hash.
-	fn submit_extrinsic(&self, xthex_prefixed: String) -> Result<Self::Hash>;
+	fn submit_extrinsic(&self, encoded_extrinsic: Vec<u8>) -> Result<Self::Hash>;
```

Comment:
I have also thought about adding a function like this, https://github.com/scs/substrate-api-client/issues/90:

```
fn send_xt<C: Encode>(xt: UncheckedExtrinsic<C>) -> Result<Self::Hash>
```

Have you considered this? But this would be an addition, and not a replacement.

Reply:
I thought about it, but this enforces UncheckedExtrinsic. I thought having it encoded still leaves the same freedom as the String one. But I agree, having one that does enforce a specific Extrinsic type would be good, as it is less error-prone.

Reply:
Yes, I like enforcing these things. This is what rust is about for me! :D

Reply:
Aye, Rust is, but Substrate is less so. Respective our client. In case the Extrinsic Type is updated, our client will be rendered useless until updated. So having both is a good compromise I think.

Reply:
Substrate enforces a lot too IMO.

Whatever, I agree having both should be the best.